Services/Administration/OJS.Services.Administration.Business/Validation/Helpers/Implementations/ContestsValidationHelper.cs

```diff
@@ -33,4 +41,32 @@ public async Task<ValidationResult> ValidatePermissionsOfCurrentUser(int? contes
             ? ValidationResult.Valid()
             : ValidationResult.Invalid("You don't not have permissions for this contest");
     }
+
+    public async Task<ValidationResult> ValidateActiveContestCannotEditDurationTypeOnEdit(
+        Contest existingContest,
+        Contest newContest)
+    {
+        var isActive = await this.activityService.IsActive(existingContest.Map<ContestForActivityServiceModel>());
+
+        if (isActive &&
+            (existingContest.Duration != newContest.Duration ||
+             existingContest.Type != newContest.Type))
+        {
+            return ValidationResult.Invalid(Resource.ActiveContestCannotEditDurationType);
+        }
+
+        return ValidationResult.Valid();
+    }
+
+    public async Task<ValidationResult> ValidateContestIsNotActiveOnDelete(Contest contest)
```

Comment:
This method does not validate anything specific for delete. just contest activity.

Reply:
The method was directly transferred from ContestValidationFactory so i kept the original name. Changing it to ValidateContestIsNotActive